stylus/ui-components/tabs.styl

```diff
@@ -6,7 +6,7 @@ $tabs-base
 
     .coz-tab
         display        inline-block
-        padding        10px 15px
+        padding        .625rem 1rem
```

Comment:
I maybe* get rid of `em` in every other context but font-size and use `rem` instead in the near future. `em` is way too much a pain in the ass even for me and I don't want to make things complicated for everyone.

* that means i strongly want to do that.